elated-egrets/bangazon-api
# removed migrations from git tracking

## DESCRIPTION
removed migrations folder from git tracking
## How to pull down
Commands

```
git fetch --all
git pull remove-migrations-from-tracking
git checkout remove-migrations-from-tracking
python manage.py runserver
```

## Steps to test
1. check to make sure your migrations folder is no longer being tracked by re running migrations and seeing if git cares

## What you changed
1. removed migrations folder from tracking and added it to gitignore

## What was added to read me
n/a

## Final Checklist
Before merging.
1. Has each file changed been reviewed?
1. Has the readme been updated to reflect changes made?
1. Has a teammate approved the PR?

Then Ship It!
